src/converse-core.js

```diff
@@ -335,6 +335,7 @@
             sid: undefined,
             sounds_path: '/sounds/',
             storage: 'session',
+            synchronize_presence: true, // Set to false to not sync with other clients or with resource name of the particular client that it should synchronize with
             use_vcards: true,
```

Comment:
Please make this `false` by default. Ideally, existing users who upgrade their version of converse.js shouldn't have to set a new option variable to maintain existing behavior.